src/main/java/javaserver/Request.java

```diff
@@ -0,0 +1,40 @@
+package javaserver;
+
+import java.util.Arrays;
+
+public class Request {
+	String method;
+	String URI;
+
+	Request(String method, String URI) {
+		this.method = method;
+		this.URI = URI;
+	}
+
+	public String getMethod() {
+		return method;
+	}
+
+	public String getURI() {
+		return URI;
+	}
+
+	public boolean isOK() {
```

Comment:
I feel like `Request` knows too much. It should just be an object that you're building from parsing it from the stream, and there should be something else that determines if the request is supported.